torch/linalg/__init__.py

```diff
@@ -272,6 +272,92 @@
             [-3.1113,  2.7381]], dtype=torch.float64)
 """)
 
+lstsq = _add_docstr(_linalg.linalg_lstsq, r"""
+torch.linalg.lstsq(input, b, cond=None, driver_name=None) -> (Tensor x, Tensor rank, Tensor s)
```

Comment:
```
torch.linalg.lstsq(input, b, cond=None, *, driver=None) -> (Tensor x, Tensor residuals, Tensor rank, Tensor s)
```

After "driver_name" is changed and made kwarg-only and residuals are added as an output.